### PR TITLE
fix: textinput handlefocus and styles

### PR DESCRIPTION
**Pull request by hoangSTS**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

https://github.com/user-attachments/assets/ece7262f-d271-4807-acfc-c5ae5dfb1bcd

### CODE REVIEW

File: src/components/TextInput/TextInput.tsx [hunk 1]
```diff
@@ -111,23 +111,26 @@ export const TextInput = forwardRef<TextInputRef, TextInputProps>(
...
             {!!isRequire && <StarText testID="test-startText"> *</StarText>}
           </Title>
         )}
         <TouchableContainer
-          style={inputContainerStyle ?? TextInputTheme.inputContainerStyle}
+          style={[TextInputTheme.inputContainerStyle, StyleSheet.flatten(inputContainerStyle)]}
           activeOpacity={1}
           onPress={handleFocus}
-          disabled={editable ?? TextInputTheme.editable}>
+          disabled={!!editable}>
```
Comment (danhmaisgt) [inline src/components/TextInput/TextInput.tsx]: I think it should be `disabled={!editable}>`

Comment (hoangSTS) [reply src/components/TextInput/TextInput.tsx]: Great! You got the point

**Review by hangnguyensaigontech — APPROVED** [review]

File: example/components/TextInputDemo/InteractivePlayground.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+          </TouchableOpacity>
+
+          <TouchableOpacity
+            style={[demoStyles.controlButton, hasSuccess && demoStyles.controlButtonSecondary]}
+            onPress={toggleSuccess}
+          >
+            <Text style={demoStyles.controlButtonText}>
+              {hasSuccess ? 'Remove Success' : 'Add Success'}
+            </Text>
+          </TouchableOpacity>
```
Comment (danhmaisgt) [inline example/components/TextInputDemo/InteractivePlayground.tsx]: Do we need the this? I see it's just rendering the text

Comment (hoangSTS) [reply example/components/TextInputDemo/InteractivePlayground.tsx]: I think it's a useless button as well

**Review by danhmaisgt — APPROVED** [review]

**Review by loc-nguyenthien — APPROVED** [review]
